pkg/webhook/resources/virtualmachine/mutator.go

```diff
@@ -215,6 +215,33 @@ func (m *vmMutator) patchResourceOvercommit(vm *kubevirtv1.VirtualMachine) ([]st
 	return patchOps, nil
 }
 
+// compute a higher default reserved memory for VM to tolerant
+// the potential more memory used by those backend components like qemu, kubevirt
+// refer: https://github.com/harvester/harvester/issues/5768
+func computeReservedMemory(mem, reservedMemory *resource.Quantity) {
```

Comment:
Do you have a benchmark or evaluation result for these computed values?
nit: it would be nice to use constants for these values for readability.

Reply:
constants update.

Benchmark, no direct data. More from community/jira ticket feedback, 1G reserved memory settled most cases even when VM's memory is about 64G, 512M is also quite good for VM with memory around 4~6G.  For specific senarios, user needs to do some tests and ajust this value accordingly.

There is an ongoing document PR to explain more details.